bogota-july-2018/javascript-calculator/Felipe Jimenez/js/script.js

```diff
@@ -0,0 +1,13 @@
+function result(){
+	var board=document.getElementById('board');
+	try {
+		var first=document.getElementById('numberLeft').value;
+		var second=document.getElementById('numberRight').value;
+		var operator=document.getElementById('operator').value;
+		var result= eval(first.concat(operator).concat(second));
```

Comment:
eval has several pitfalls and in a clasic calculator, we never display the whole operation. just the numbers